nemo/collections/tts/models/tacotron2.py

```diff
@@ -196,10 +196,12 @@ def output_types(self):
 
     @typecheck()
     def forward(self, *, tokens, token_len, audio=None, audio_len=None):
-        if audio is not None and audio_len is not None:
-            spec_target, spec_target_len = self.audio_to_melspec_precessor(audio, audio_len)
+        assert audio is not None and audio_len is not None, f"'audio' and 'audio_len' can not be None."
```

Comment:
Will we always have audio/audio_len ground truth for the forward function here?

Reply:
hmm...good catch. Let me correct that.